addons/project/models/project_task.py

```diff
@@ -327,6 +329,25 @@ def _compute_state(self):
             elif task.state not in CLOSED_STATES:
                 task.state = '01_in_progress'
 
+    @api.depends('state')
+    def _compute_is_closed(self):
+        for task in self:
+            task.is_closed = task.state in CLOSED_STATES
+
+    @api.model
```

Comment:
No need when the method is private. In fact, this decorator is useful when the method is callable via RPC, it will allow us to ignore the `self` parameter to pass as arguments. 🙂